shapeshift/web
# feat: merge cosmos get started to the stake modal

## Description
- We want to merge both `GetStarted` and `LearnMore` into the cosmos `Stake` modal
- I move both file into the Staking views, including it inside `Overview.tsx` and toggling depending if there are some staking datas
- I added a GetStarderManager file with a router in order to manage both routes
- Removed every occurrences of the GetStarted modal
- I need to use the parent history as a prop inside the GetStarted flow because we need to push to the staking route



## Notice




## Pull Request Type


## Issue (if applicable)
closes #1624


## Risk
- Breaking the cosmos flow


## Testing
- Using an account without any staking on cosmos
- Click on the opportunity row, it should open the cosmos modal on the GetStarted view
- You should be able to click on Stake or Learn more and see every learn more views
- Other cosmos features should work as expected

- Using an account with staking datas on cosmos
- Click on the opportunity row, it should open the cosmos modal on the Overview view
- Other cosmos features should work as expected



## Screenshots (if applicable)
https://www.awesomescreenshot.com/video/8758745?key=56732b8f246b8a4c0a55d04d008b019b
https://www.awesomescreenshot.com/video/8758730?key=5e673af113f06b94f32053f851bf2f0a
